build/buildTestBuildImages.sh

```diff
@@ -15,12 +15,28 @@ source $REPO_DIR/build/__functions.sh
 buildImageDebianFlavor="$1"
 
 echo
-echo Building build images for tests...
+echo "Building build images for tests..."
+
 docker build \
-    -t "$ORYXTESTS_BUILDIMAGE_REPO:latest" -f "$ORYXTESTS_BUILDIMAGE_DOCKERFILE" .
+    -t "$ORYXTESTS_BUILDIMAGE_REPO:github-actions" \
```

Comment:
Reason behind creating a restrictive permissions image for `github-actions` image is to simulate AppService environment who are going to be using this going forward.